File: rules.py
# More Rules will be added in the future
RULES = [
    {
        "description": "No words with the letter 'e'",
        "check": lambda text: 'e' not in text.lower()
    },
    {
        "description": "Must end with a question mark",
        "check": lambda text: text.strip().endswith("?")
    },
    {
        "description": "Exactly 5 words",
        "check": lambda text: len(text.strip().split()) == 5
    }
]

File: rules.js
const RULES = [
  {
    description: "Do not use the letter E",
    check: (text) => !text.toLowerCase().includes("e")
  },
  {
    description: "Only questions are allowed",
    check: (text) => text.trim().endsWith("?")
  },
  {
    description: "Start every sentence with the letter T",
    check: (text) => text.trim().toLowerCase().startsWith("t")
  }
];
